endpoints/openrtb2/auction_test.go

```diff
@@ -3232,57 +3230,6 @@ func TestParseRequestParseImpInfoError(t *testing.T) {
 	assert.Contains(t, errL[0].Error(), "echovideoattrs of type bool", "Incorrect error message")
 }
 
-func TestAuctionFirstPartyData(t *testing.T) {
```

Comment:
Do we need to delete this test? Is there another suitable E2E test for this already?

Reply:
This tests only checked fpd extraction from request. I don't think we have E2E test due to the way we mock functions. 
I'll check if I can add one. 
We have tests for all parts of FPD, but separately.

Reply:
Added!